.gitignore

```diff
@@ -2,8 +2,8 @@
 #
 # gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
 
-CourseGrab/GoogleService-Info.plist
-CourseGrab/Keys.plist
+*GoogleService-Info.plist
+*Keys.plist
```

Comment:
can we also explicitly specify the locations of these files rather than using `*`. it'll make it clear where other developers should be placing these files in their projects